funnel/templates/contacts.html.jinja2

```diff
@@ -0,0 +1,84 @@
+{% extends "layout.html.jinja2" %}
+{% block title %}{% trans %}Contacts{% endtrans %}{% endblock %}
+
+{% block content %}
+  {% raw %}
+    <div class="grid" id="scan-badge-wrapper">
+      <script id='scan-badge-template' type='text/ractive'>
+        <div class="grid__col-xs-12 grid__col-sm-6">
+        {{#if video}}
+          <h4 class="mui--text-subhead">Scan attendee badge</h4>
+          <p><button type="button" class="mui-btn mui-btn--small mui-btn--raised mui-btn--accent" on-click="setupVideo(event)">Restart QR reader</button></p>
+        {{elseif error}}
+          <p class="mui--text-danger">{{ error }}</p>
+        {{/if}}
+        <canvas id="qrreader-canvas"></canvas>
+
+        {{#if showModal}}
+        <div id="status-msg" class="modal" tabindex="-1" role="dialog">
+          <div class="modal__header">
+            <a class="modal__close" href="javascript:void(0);" on-click="closeModal(event)"><i class="material-icons">clear</i></a>
+            <h3 class="mui--text-title">Contact exchange</h3>
```

Comment:
"Scanned contacts"

It is not an exchange if the other party doesn't also receive contacts. We haven't built this ability yet.